src/Nncase.Evaluator/Math/Binary.cs

```diff
@@ -157,13 +165,24 @@ private IValue Ort_compute(Binary binary, Tensor lhs, Tensor rhs)
     {
         var a = lhs.ToOrtTensor();
         var b = rhs.ToOrtTensor();
+        Func<OrtKISharp.Tensor, OrtKISharp.Tensor, OrtKISharp.Tensor> mod = (a, b) =>
+        {
+            if (DataTypes.IsFloat(a.DataType.ToDataType()) && DataTypes.IsFloat(b.DataType.ToDataType()))
+            {
+                return OrtKI.Mod(a, b, 1);
```

Comment:
You can convert the result of the condition to int and pass it directly to OrtKI.